web-ui/src/main/resources/catalog/components/catalog/CatalogService.js

```diff
@@ -761,9 +761,25 @@
         getUuid: function () {
           return this.uuid;
         },
-        isPublished: function () {
+        isPublished: function (pubOption) {
+          if (pubOption) {
+            if (pubOption.publicationGroup === "all") {
```

Comment:
If we want to make it work for any groups, maybe we could do:
* indexing / for all groups add a field `isPublishedTo{groupName}`
* client side / check if those fields are set for the pubOptions.
?
This could reduce repetitions here and in indexing and make it a bit more generic.
BTW, we don't need it right now, so we can keep this for later.